### ISSUE TITLE
[question] Is there an online man page / documentation for CLI usage?

### ISSUE THREAD

**Issue by toraritte** [DeterminateSystems/nix-installer#766]
TL;DR Couldn't find a CLI man page or similar docs online.

These are the places I looked:

1. `google search` [determinate systems nix installer man page documentation - Google Search](https://www.google.com/search?q=determinate+systems+nix+installer+man+page+documentation&sca_esv=587665533&sxsrf=AM9HkKmtNQ83VVhVHs57Pg9XtBhB_ppgGA%3A1701697353200&ei=SddtZb_nC5-JptQP5LOl8AE&ved=0ahUKEwj_r4LS9PWCAxWfhIkEHeRZCR4Q4dUDCBA&uact=5&oq=determinate+systems+nix+installer+man+page+documentation&gs_lp=Egxnd3Mtd2l6LXNlcnAiOGRldGVybWluYXRlIHN5c3RlbXMgbml4IGluc3RhbGxlciBtYW4gcGFnZSBkb2N1bWVudGF0aW9uMgUQIRigAUjgG1CqBVigGXACeAGQAQCYAbIBoAHpDqoBBDAuMTW4AQPIAQD4AQHCAgoQABhHGNYEGLADwgIFECEYqwLiAwQYACBBiAYBkAYI&sclient=gws-wiz-serp)
2. `determinate.systems` [Introducing the Determinate Nix Installer](https://determinate.systems/posts/determinate-nix-installer)
3. `zero-to-nix.com` [The Determinate Nix Installer](https://zero-to-nix.com/concepts/nix-installer)
4.  `github` [DeterminateSystems/nix-installer: Install Nix and flakes with the fast and reliable Determinate Nix Installer, with over a million installs.](https://github.com/DeterminateSystems/nix-installer)

Planning to use the installer in a script, so was looking for a way to silence the prompts. Found the answer eventually in link 2. ([`--no-confirm`](https://determinate.systems/posts/determinate-nix-installer#a-flexible-installer)), but had to dig through all of the links. (Similar options for other popular CLI tools are usually along the lines of `--quiet` or `--silent`, but then [neither the official nor the `nix.dev` installer use them and opt for `--yes` instead](https://unix.stackexchange.com/a/736235/85131).)

Please feel free to close this if there is indeed no such online resource; was only asking to make sure I didn't overlook something. Hands down this is the best Nix installer I used thus far, so thank you for your hard work!

**Comment by Hoverbear** [comment]
No online man page, but you can do

```
curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | sh -s -- --help
```
